src/main/java/com/google/gwt/site/webapp/client/GWTProjectEntryPoint.java

```diff
@@ -443,23 +492,37 @@ private boolean isHomePage(String path) {
   }
 
   private void styleHomepage() {
-    final int windowHeight = $(window).height();
-    int sectionHeight = $("#letsbegin").height();
+    final int windowHeight = window.innerHeight;
+    int sectionHeight = Js.<HTMLElement>uncheckedCast(document.querySelector("#letsbegin")).offsetHeight;
 
     if (windowHeight > sectionHeight) {
-      $(".home section").each(new Function() {
-        @Override
-        public void f() {
-          $(this)
-              .css("height", windowHeight + "px")
-              .css("padding", "0");
-          GQuery container = $(this).find(".container");
-          container.css("padding-top", (windowHeight - container.height()) / 2 + "px");
-        }
+      forEach(".home section", element -> {
+        element.style.setProperty("height", windowHeight + "px");
+        element.style.setProperty("padding", "0");
+        HTMLElement container = Js.uncheckedCast(element.querySelector(".container"));
+        container.style.setProperty("padding-top", (windowHeight - container.offsetHeight) / 2 + "px");
       });
     }
   }
 
+  private void forEach(String selector, Consumer<HTMLElement> consumer) {
```

Comment:
This certainly does the job, but two thoughts I want to suggest (for a follow up patch, possibly written by someone else, just to discuss now):
 * Make it more Java: switch this over to a stream of some kinda, so calls to it can chain the .forEach(consumer), or can add filters and .findFirst() (rather than the array[0] trick). This makes it clearer that you can use plain Java here.
 * Make it more JS: make a specific lambda type marked with `@JsFunction` instead of java's Consumer. This will let the compiler skip attaching the java properties to the lambda and make smaller, clearer JS output. This makes the output a little nicer to read for someone interested in writing clean JS in plain Java.

Reply:
I think "more JS" is better, so I went with that one.